flycheck.el

```diff
@@ -3395,9 +3408,37 @@ Return the created overlay."
       (setf (overlay-get overlay 'before-string)
             (flycheck-error-level-make-fringe-icon
              level flycheck-indication-mode)))
-    (setf (overlay-get overlay 'help-echo) (flycheck-error-message err))
+    (setf (overlay-get overlay 'help-echo) #'flycheck-help-echo-helper)
     overlay))
 
+(defun flycheck-help-echo-helper (window object pos)
```

Comment:
I'd just call this function `flycheck-help-echo`.  “-helper” seems a little redundant to me.